package.json

```
@@ -11,46 +11,50 @@
     "scripts/validate-dist"
   ],
   "scripts": {
-    "build-storybook": "build-storybook -o dist",
-    "build:all": "ts-node ./scripts/build-all",
-    "build:ci": "NODE_ENV=production build-storybook -o dist",
-    "clean": "rm -rf dist",
-    "clean:all": "ts-node ./scripts/clean-all",
+    "build": "npm-run-all build:src -p build:kitchen build:foundations -s build:storybook",
+    "build:foundations": "yarn workspace @guardian/source-foundations build",
+    "build:kitchen": "yarn workspace @guardian/source-react-components-development-kitchen build",
+    "build:src": "ts-node ./scripts/build-src",
+    "build:storybook": "build-storybook -o dist",
+    "build:storybook:ci": "NODE_ENV=production build-storybook -o dist",
+    "clean": "npm-run-all clean:*",
+    "clean:foundations": "yarn workspace @guardian/source-foundations clean",
+    "clean:kitchen": "yarn workspace @guardian/source-react-components-development-kitchen clean",
+    "clean:root": "rm -rf dist",
+    "clean:src": "ts-node ./scripts/clean-src",
     "fix": "yarn lint:js --fix",
-    "foundations": "yarn workspace @guardian/source-foundations",
     "get-usage": "ts-node ./scripts/get-usage",
-    "kitchen": "yarn workspace @guardian/source-react-components-development-kitchen",
     "lint": "yarn lint:js && yarn lint:styles",
     "lint:js": "eslint . --ext .ts,.tsx,.js",
     "lint:styles": "stylelint src/core/**/styles.ts",
     "prepare": "husky install",
     "publish:release": "ts-node ./scripts/publish-release",
-    "release:major:all": "yarn build:all && yarn validate && yarn verbump:major:all && yarn publish:release",
-    "release:minor:all": "yarn build:all && yarn validate && yarn verbump:minor:all && yarn publish:release",
-    "release:patch:all": "yarn build:all && yarn validate && yarn verbump:patch:all && yarn publish:release",
-    "release:prerelease-bump:all": "yarn validate && yarn verbump:prerelease:all && yarn publish:release",
-    "release:prerelease:all": "yarn validate && yarn publish:release",
+    "release:major:src": "yarn build:src && yarn validate && yarn verbump:major:src && yarn publish:release",
+    "release:minor:src": "yarn build:src && yarn validate && yarn verbump:minor:src && yarn publish:release",
+    "release:patch:src": "yarn build:src && yarn validate && yarn verbump:patch:src && yarn publish:release",
+    "release:prerelease-bump:src": "yarn validate && yarn verbump:prerelease:src && yarn publish:release",
+    "release:prerelease:src": "yarn validate && yarn publish:release",
     "storybook": "start-storybook -p 6006",
-    "tag:beta:all": "ts-node ./scripts/tag-beta-all",
+    "tag:beta:src": "ts-node ./scripts/tag-beta-src",
     "test": "jest",
     "test:coverage": "jest --coverage",
     "tsc": "tsc",
     "validate": "yarn tsc && yarn lint && yarn test",
     "validate-dist": "yarn build:all && yarn workspace validate-dist validate",
```

This should just be `yarn build` now, right?

```suggestion
    "validate-dist": "yarn build && yarn workspace validate-dist validate",
```

---

Oh yes good spot